crypto/crypto.go

```diff
@@ -233,7 +233,7 @@ func aesCBCDecrypt(key []byte, cipherText []byte, iv []byte) (plainText []byte,
 	decrypter.CryptBlocks(paddedPlainText, cipherText)
 	plainText = PKCS7Unpad(paddedPlainText)
 	if plainText == nil {
-		err = errors.New("Decryption failed: PKCS7Unpad failed after decryption")
+		err = errors.New("Decryption failed: PKCS7Unpad failed after AES decryption")
```

Comment:
It's good to differentiate unpadding error from MAC mismatch error when decrypting. I think it's fine to display this to command-line users, since they will be a tech-savvy demographic.